### PR TITLE
Add UG details for import-export

### LINKED ISSUE #300: [PE-D][Tester C] Functionality Bug: Cannot add Telegram via CSV file
I am unable to add the telegram handle of a student via the CSV style due to `"@"` being used in Excel for functions and commands. As such, I cannot add a new student from the CSV file.

This issue persists even if I try to click off etc. (Due to Excel)

Edit: Its a known issue, but I had to scroll all the way to the bottom of the UG to see it

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: lrongyi/ped#8

### LINKED ISSUE #354: Export and import does not handle ~ correctly
By right, `~` refers to the home folder. but now it creates a folder named `~` instead.

### PR DESCRIPTION
**Pull request by tohjh**
closes #354 
closes #300

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -753,19 +762,28 @@ Exports contacts based on contacts and their details stored in KonTActs.
...
 
 <md>**Examples:**</md>
 
 * `export path/data/group12.csv` will export the contacts in KonTActs as a csv file (group12.csv at the 
-relative path given)
-* Alternatively, `ex p/data/group12.csv` will do the exact same. It uses the shortcut command and parameter. 
+RELATIVE path given).
+* Alternatively, `ex p/data/group12.csv` will do the exact same. It uses the shortcut command and parameter.
+* `export path//data/group12.csv` will export the contacts in KonTActs as a csv file (group12.csv at the
+ABSOLUTE path given - absolute paths specify the exact location in the file system, starting from the root directory,
```
Comment (SwaminathanViswa) [inline docs/UserGuide.md:780]: Maybe break the long sentence into 2 for better reading. "... ABSOLUTE path given. Absolute paths specify..."

**Review by SwaminathanViswa — COMMENTED** [review]
LGTM! just a small thought for better understandability.

**Review by Incogdino — COMMENTED** [review]
LGTM, maybe can add a link for more clarity too

**Review by Incogdino — APPROVED** [review]
LGTM

**Pull request merged**